tests/integration/helpers.py

```diff
@@ -302,9 +302,9 @@ def tempo_worker_charm_and_channel_and_resources():
     """Tempo worker charm used for integration testing.
 
     Build once per session and reuse it in all integration tests to save some minutes/hours.
-    You can also set `TEMPO_WORKER_CHARM` env variable to use an already existing built charm.
+    You can also set `WORKER_CHARM_PATH` env variable to use an already existing built charm.
     """
-    if path_from_env := os.getenv("TEMPO_WORKER_CHARM"):
+    if path_from_env := os.getenv("WORKER_CHARM_PATH"):
```

Comment:
Nothing in CI is setting this, though I'm not sure what your worker CI does

Reply:
it's just for local testing